src/core/utils.js

```diff
@@ -153,3 +158,21 @@ export function browserBase64Decode(str) {
     return '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2);
   }).join(''));
 }
+
+export function apiAddonType(addonType) {
+  if (!Object.prototype.hasOwnProperty.call(
```

Comment:
Yeah I was pretty surprised by that. I guess I see the sense but it's weird.

I tried the `const` thing, but it's less clear and also it didn't work. So I'll leave it as-is and be like "oh JavaScript".

![wat](https://cloud.githubusercontent.com/assets/90871/21949728/7939a09e-d9ec-11e6-8375-9d7fd018afce.jpg)